janitor/chemistry.py

```diff
@@ -57,13 +57,23 @@
         GetMACCSKeysFingerprint,
     )
 except ImportError:
-    import_message("chemistry", "rdkit", "conda install -c rdkit rdkit")
+    import_message(
+        submodule="chemistry",
+        package="rdkit",
+        conda_channel="rdkit",
```

Comment:
Since we're at it, let's also change this to conda-forge. I've tested both the `rdkit` and `conda-forge` channel builds, and I'm confident both of them work equivalently.
```suggestion
        conda_channel="conda-forge",
```